### DISCUSSION TITLE
Reviewers: Something went wrong when executing your query

### DISCUSSION THREAD

**Discussion by VictorXefi** [community/community#63817] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

I am executing the following GraphQL query:
`{
  node(id: "PVTI_lAHOBzJYkM4AUHBOzgIkOMI") {
    ... on ProjectV2Item {
      fieldValueByName(name: "Reviewers") {
        ... on ProjectV2ItemFieldReviewerValue {
          reviewers(first: 100) {
            edges {
              node {
                ... on User {
                  login
                }
              }
            }
          }
        }
      }
    }
  }
}`

And receive the following error:
`{
  "errors": [
    {
      "message": "Something went wrong while executing your query. Please include `81A3:68E0:1467D12:2A25129:64DB82A8` when reporting this issue."
    }
  ]
}`

Is this a bug or can you not query reviewers?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @VictorXefi ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
